README.md

```diff
@@ -37,6 +37,15 @@ may file one [here](https://github.com/NVIDIA/spark-rapids/issues/new/choose).
 
 ## Build
 
+There are two types of branches in this repository:
+
+* `branch-[version]`: are development branches which can change often. Note that we merge into 
+  the branch with the greatest version number, as that is our default branch.
+
+* `main`: is the branch with the latest released code, and the version tag (i.e. `v0.1.0`)
+  is held here. `main` will change with new releases, but otherwise it should not change with
+  every pull request merged, and should be a more stable branch.
```

Comment:
Nit: is a more stable branch , perhaps ? Instead if should be.

Reply:
@kuhushukla, please take another look. I made it slightly different.